<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EggSettings</class>
 <widget class="QWidget" name="EggSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>222</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="labelParentAIVs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Parent A</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labelParentBIVs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Parent B</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="labelParentA">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Parent A</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="labelParentB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Parent B</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelHP">
     <property name="text">
      <string>HP</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBoxParentAHP">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinBoxParentBHP">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="labelAbility">
     <property name="text">
      <string>Ability</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="comboBoxParentAAbility"/>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="comboBoxParentBAbility"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelAtk">
     <property name="text">
      <string>Atk</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBoxParentAAtk">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="spinBoxParentBAtk">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="labelGender">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="comboBoxParentAGender"/>
   </item>
   <item row="2" column="5">
    <widget class="QComboBox" name="comboBoxParentBGender"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelDef">
     <property name="text">
      <string>Def</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinBoxParentADef">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spinBoxParentBDef">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="labelItem">
     <property name="text">
      <string>Item</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="ComboBox" name="comboBoxParentAItem"/>
   </item>
   <item row="3" column="5">
    <widget class="ComboBox" name="comboBoxParentBItem"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelSpA">
     <property name="text">
      <string>SpA</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinBoxParentASpA">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="spinBoxParentBSpA">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="labelNature">
     <property name="text">
      <string>Nature</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QComboBox" name="comboBoxParentANature"/>
   </item>
   <item row="4" column="5">
    <widget class="QComboBox" name="comboBoxParentBNature"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelSpD">
     <property name="text">
      <string>SpD</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spinBoxParentASpD">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="spinBoxParentBSpD">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="labelEggSpecie">
     <property name="text">
      <string>Egg Specie</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="ComboBox" name="comboBoxEggSpecie"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelSpe">
     <property name="text">
      <string>Spe</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="spinBoxParentASpe">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="spinBoxParentBSpe">
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QCheckBox" name="checkBoxMasuda">
     <property name="text">
      <string>Masuda</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="2">
    <widget class="QCheckBox" name="checkBoxShowInheritance">
     <property name="text">
      <string>Show Inheritance</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/ComboBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
